MSBuildProjectTools.sln

```diff
@@ -33,8 +33,6 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "LanguageServer.SemanticMode
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "LanguageServer.Common", "src\LanguageServer.Common\LanguageServer.Common.csproj", "{5AD7AEFF-4E5C-474C-8BD6-DEBD354A803C}"
 EndProject
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "LanguageServer.TaskReflection", "src\LanguageServer.TaskReflection\LanguageServer.TaskReflection.csproj", "{C594F3C8-8E90-4AE2-B75C-DB17AC652735}"
-EndProject
```

Comment:
It seems like you removed project reference from the solution, but didn't delete actual project files. Remove this files before merging pls